platforms/hyperledger-fabric/configuration/roles/create/chaincode/commit/tasks/write.yaml

```diff
@@ -0,0 +1,8 @@
+---
+# This task writes the certificates of all the approving organizations to the vault so that the creator organization can use it to commit.
+- name: Copy organization level certificates for orgs {{ endorser.name }}
+  shell: |
+    vault write /secret/crypto/peerOrganizations/{{ namespace }}/endorsers/{{ endorser.name }}/users/admin/msp cacerts="$(cat ./build/crypto-config/peerOrganizations/{{ endorser.name }}-net/users/Admin@{{ endorser.name }}-net/msp/cacerts/ca-{{ endorser.name }}-net-7054.pem)"
```

Comment:
will the certificates be there in the given path if you are running only the chaincode/commit tasks?

Reply:
So These certificates of all the approving organization are being fetched from the build folder, that is required for the chaincode commit.
Thats why the delete build folder was shifter to the end for chaincode-install-instantiate.yaml playbook

Reply:
This approach is wrong because the certificates wont be there in the "build" folder, the path for these certificates should be taken from network.yaml